util/deploy-to-maven-central.sh

```diff
@@ -50,13 +50,15 @@ for generated_pom_file in dagger*pom.xml; do
   rm "${generated_pom_file}.asc"
 done
 
-release_branch="release_${version_name}"
-git checkout -b "${release_branch}"
+# Switch to a detached HEAD
+git checkout $(git rev-parse --verify HEAD)
```

Comment:
Fwiw, I learned today about the `--detach` flag to `git checkout`(which would _a priori_ remove the need for `git rev-parse`)

Reply:
Cool, just incorporated that into the internal commit for this.